subsys/net/openthread/rpc/Kconfig

```diff
@@ -0,0 +1,49 @@
+#
+#
+# Copyright (c) 2024 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menuconfig OPENTHREAD_RPC
+	bool "OpenThread over RPC [EXPERIMENTAL]"
+	select NRF_RPC
+	select NRF_RPC_CBOR
+	select EXPERIMENTAL
+	help
+	  Enables OpenThread serialization over RPC
+
+if OPENTHREAD_RPC
+
+choice OPENTHREAD_RPC_ROLE_CHOICE
+	prompt "OpenThread over RPC role selection"
+	default OPENTHREAD_RPC_CLIENT
+	help
+	  Selects the device role for OpenThread over RPC. The default role is
+	  a client role.
+
+config OPENTHREAD_RPC_CLIENT
+	bool "OpenThread over RPC client"
+	depends on NETWORKING
```

Comment:
Do we plan to keep that dependency in the future? Avoiding it would enable integrations in IP-stack-less environments, i.e. Customer would be able to rely on L7 API, i.e. COAP

Reply:
Only `if_client` file depends on it, so we can compile it conditionally based on `NETWORKING` instead.

Reply:
This can be mitigated for future Since this is currently only option it has to stay. You can rais a Jira for reference.